EIPS/eip-3382.md

```diff
@@ -0,0 +1,50 @@
+---
+eip: 3382
+title: Hardcoded Block Gas Limit
+author: Philippe Castonguay (@PhABC)
+discussions-to: https://ethereum-magicians.org/t/eip-3382-hardcoded-gas-limit
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-03-13
+---
+
+## Simple Summary
+
+Hardcode the block gas limit (known as *block gas target* under [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md)) to `12,500,000` gas per block.
```

Comment:
Links to external EIPs should be relative (`[EIP-1559](./eip-1559.md)`).  That being said, this is not really a detail relevant to the simple summary.  The simple summary should be just a short one liner that says what the EIP is.
```suggestion
Hardcode the block gas target to `12,500,000` gas per block.
```